Dockerfile.prod

```diff
@@ -0,0 +1,28 @@
+FROM python:3.12.7-alpine3.20@sha256:edd1d8559c585e1e9a9b79de44ac27f8ac32cb0c7323e112ae6870ceeecd8dbf AS builder
```

Comment:
You can probably get away with one stage as I don't think it would make a massive difference in this case.